cmd/tcl/testworkflow-toolkit/commands/kill.go

```diff
@@ -96,6 +96,39 @@ func NewKillCmd() *cobra.Command {
 					}
 				}
 
+				for _, id := range ids {
+					service, index := spawn.GetServiceByResourceId(id)
+					count := index + 1
+					if services[service] > count {
+						count = services[service]
+					}
+
+					log := spawn.CreateLogger(service, "", index, count)
+					notifications := spawn.ExecutionWorker().Notifications(context.Background(), id, executionworkertypes.NotificationsOptions{})
+					if notifications.Err() != nil {
+						log("error", "failed to connect to the service", notifications.Err().Error())
+						continue
+					}
+
+					for l := range notifications.Channel() {
+						if l.Result == nil || l.Result.Status == nil {
+							continue
+						}
```

Comment:
This will be endless loop if the service is long-running

Reply:
Added NoFollow